core/src/main/scala/io/udash/generator/plugins/scalacss/ScalaCSSDemosPlugin.scala

```diff
@@ -131,49 +256,1050 @@ object ScalaCSSDemosPlugin extends GeneratorPlugin with SBTProjectFiles with Fro
          |import scalacss.Defaults._
          |import scalacss.ScalatagsCss._
          |import scalatags.JsDom._
-         |import scalatags.JsDom.all._
+         |import ${settings.rootPackage.mkPackage()}.${settings.stylesSubPackage.mkPackage()}.GlobalStyles
          |import ${settings.rootPackage.mkPackage()}.${settings.stylesSubPackage.mkPackage()}.DemoStyles
-         |jQ(appRoot.get).addClass(DemoStyles.container.className.value)
-         |jQ(DemoStyles.render[TypedTag[org.scalajs.dom.raw.HTMLStyleElement]].render).insertBefore(appRoot.get)""".stripMargin)
+         |import ${settings.rootPackage.mkPackage()}.${settings.stylesSubPackage.mkPackage()}.partials.FooterStyles
+         |import ${settings.rootPackage.mkPackage()}.${settings.stylesSubPackage.mkPackage()}.partials.HeaderStyles
+         |jQ(GlobalStyles.render[TypedTag[org.scalajs.dom.raw.HTMLStyleElement]].render).insertBefore(appRoot.get)
+         |jQ(DemoStyles.render[TypedTag[org.scalajs.dom.raw.HTMLStyleElement]].render).insertBefore(appRoot.get)
+         |jQ(FooterStyles.render[TypedTag[org.scalajs.dom.raw.HTMLStyleElement]].render).insertBefore(appRoot.get)
+         |jQ(HeaderStyles.render[TypedTag[org.scalajs.dom.raw.HTMLStyleElement]].render).insertBefore(appRoot.get)""".stripMargin)
+
+    writeFile(globalStylesScala)(
+      s"""package ${settings.rootPackage.mkPackage()}.${settings.stylesSubPackage.mkPackage()}
+          |
+          |import com.example.styles.constants.StyleConstants
+          |import com.example.styles.fonts.{FontStyle, FontWeight, UdashFonts}
+          |import com.example.styles.utils.{MediaQueries, StyleUtils}
```

Comment:
Replace `com.example` with `${settings.rootPackage.mkPackage()}`, please. Generated code does not compile, when you provide different root package.